website/docs/concepts/reading.mdx

```diff
@@ -202,7 +208,7 @@ like inside `onPressed` or an [ElevatedButton]. Nor should it be used
 inside `initState` and other [State] life-cycles.
 :::
 
-### Using ref.read to obtain the state of a provider once
+### Using ref.read to obtain the state of a provider at the current moment
```

Comment:
I don't like "at the current moment" here. watch is also "at the current moment" after-all